File: app/src/main/java/me/echeung/moemoekyun/client/api/library/Kpop.java
package me.echeung.moemoekyun.client.api.library;

public class Kpop extends Library {

    public static final Library INSTANCE = new Kpop();

    public static final String NAME = "kpop";

    private Kpop() {
        super(NAME, "wss://listen.moe/kpop/gateway", "https://listen.moe/kpop/fallback");
    }

}

File: app/src/main/java/me/echeung/moemoekyun/client/api/library/Kpop.kt
package me.echeung.moemoekyun.client.api.library

class Kpop private constructor() : Library(
        NAME,
        "wss://listen.moe/kpop/gateway_v2",
        "https://listen.moe/kpop/fallback") {
    companion object {
        const val NAME = "kpop"

        val INSTANCE: Library = Kpop()
    }
}
